src/global/shared-types/img.types.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * This interface is used to specify a path to an image,
+ * along with related properties, like alt text.
+ * @public
+ */
+export interface Img {
```

Comment:
should the interface be called `Img` or `Image`?

The HTML element would be an `<img />` so in this sense, the other attributes such as `src` and `alt` would be more meaningful for the consumer perhaps.

Reply:
It should be `Image` :smile: We should avoid abbreviations